Generator/Sources/NeedleFramework/Execution/SequenceExecutor.swift

```diff
@@ -122,6 +122,45 @@ class SequenceExecutorImpl: SequenceExecutor {
     }
 }
 
+/// Executor of sequences of tasks using the main queue.
+/// This exists only to assist with debugging
+///
+/// - seeAlso: `SequencedTask`.
+class SequenceExecutorSerialImpl: SequenceExecutor {
```

Comment:
Can we also name this `SerialSequenceExecutor`?

Reply:
done